crates/runtime/benches/setup/mod.rs

```diff
@@ -128,3 +121,18 @@ fn get_branch_name() -> String {
             |output| String::from_utf8_lossy(&output.stdout).trim().to_string(),
         )
 }
+
+pub trait BenchAppBuilder {
+    fn build_app(&self, upload_results_dataset: &Option<String>) -> App;
+    fn make_dataset(&self, path: &str, name: &str) -> Dataset;
+
+    fn make_rw_dataset(&self, path: &str, name: &str, dataconnector: DataConnector) -> Dataset {
```

Comment:
Let's not have a separate table per data connector - we can add a new column to the benchmark results for the data connector.